### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by srz-zumix**
Copying files over from https://github.com/srz-zumix/reviewdog-cpplint

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Reviewdog
+on: [push, pull_request]
```
Comment (haya14busa) [inline .github/workflows/reviewdog.yml]: `push` event doesn't work with github-pr-review reporter, can you remove it?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-alpine
+# styleguide cpplint not support python3
```
Comment (haya14busa) [inline Dockerfile]: Can you remove this comment?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# FROM python:2.7-alpine
+
+RUN apk --no-cache --update add git \
+    && rm -rf /var/cache/apk/*
+
+RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b /usr/local/bin/
+
+# RUN	git clone https://github.com/google/styleguide.git cpplint \
```
Comment (haya14busa) [inline Dockerfile]: Can you remove this comment?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,57 @@
...
+Optional. List of file list arguments to send to cpplint.
+Default is `--recursive .`.
+
+## Example Usage
+
+### [.github/workflows/reviewdog.yml](.github/workflows/reviewdog.yml)
+
+```yml
+name: Reviewdog
+on: [push, pull_request]
```
Comment (haya14busa) [inline README.md]: remove `push`

File: .github/workflows/reviewdog.yml (hunk 1 above)
Comment (srz-zumix) [reply .github/workflows/reviewdog.yml]: fixed: 5f26ce8

File: README.md (hunk 4 above)
Comment (srz-zumix) [reply README.md]: fixed: 5f26ce8

File: Dockerfile (hunk 2 above)
Comment (srz-zumix) [reply Dockerfile]: fixed: b3f9234

File: Dockerfile (hunk 3 above)
Comment (srz-zumix) [reply Dockerfile]: fixed: b3f9234

**Comment by srz-zumix** [comment]
Thank you for reviewing!

**Pull request merged**